### PR TITLE
[release]Linux/SQL project

### PR DESCRIPTION
**Pull request by AdityaSharma5**

### CODE REVIEW

File: linux_sql/README.md [hunk 1]
```diff
@@ -1,2 +1,57 @@
 # Linux Cluster Monitoring Agent
 This project is under development. Since this project follows the GitFlow, the final work will be merged to the master branch after Team Code Team.
+## Introduction
```
Comment (jarviscanada) [inline linux_sql/README.md:3]: please fix markdown syntax 


https://github.com/jarviscanada/jarvis_data_eng_Aditya/blob/179d1f7ac7b27c22a21a041e41b75b3d05441687/linux_sql/README.md

File: linux_sql/scripts/host_info.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+cpu_mhz=$(echo "$lscpu_out" | egrep "^CPU MHz:" | awk '{print $3}' | xargs)
+l2_cache=$(echo "$lscpu_out" | egrep "^L2 cache:" | awk '{print $3}' | grep -o -E '[0-9]+' | awk '{print $1}' | xargs)
+total_mem=$(echo "$(cat /proc/meminfo)" | egrep "^MemTotal:" | awk '{print $2}' | xargs)
+timestamp=$(echo "$(vmstat -t)"  | egrep "1" | awk '{print $18, $19}' | xargs)
+
+
+
+insert_stmt="INSERT INTO host_info (hostname, cpu_number, cpu_architecture, cpu_model, cpu_mhz, L2_cache, total_mem, "timestamp") VALUES ('${hostname}',${cpu_number},'${cpu_architecture}', '${cpu_model}', ${cpu_mhz}, ${l2_cache}, ${total_mem}, '${timestamp}');"
+
+psql -h "$psql_host" -p "$psql_port" -U "$psql_user" -d "$db_name" -c "$insert_stmt"
```
Comment (jarviscanada) [inline linux_sql/scripts/host_info.sh]: exit $?

Comment (jarviscanada) [reply linux_sql/scripts/host_info.sh]: fix this for all shell scripts

File: linux_sql/sql/queries.sql [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+GROUP BY c.cpu_number, c.host_id, c.total_mem;
+
+
+SELECT
+    u.host_id as host_id,
+    i.hostname as host_name,
+    date_trunc('hour', u."timestamp") + date_part('minute', u."timestamp")::int / 5 * interval '5 min' as times,
+    (((i.total_mem-u.memory_free)*100/i.total_mem)) as average_mem
+FROM host_usage u, host_info i
+WHERE u.host_id = i.id;
```
Comment (jarviscanada) [inline linux_sql/sql/queries.sql]: this won't work. you need to use either group by or window function

**Review by jarviscanada — CHANGES_REQUESTED** [review]
fix and request for review again

File: linux_sql/README.md (hunk 1 above)
Comment (AdityaSharma5) [reply linux_sql/README.md:3]: fixed

File: linux_sql/scripts/host_info.sh (hunk 2 above)
Comment (AdityaSharma5) [reply linux_sql/scripts/host_info.sh]: fixed

**Pull request closed without merging**